admin_tests/nodes/test_serializers.py

```diff
@@ -1,4 +1,4 @@
-from nose.tools import *  # flake8: noqa
+from nose.tools import *  # noqa:
```

Comment:
We should always pass a specific rule with noqa, e.g.

```suggestion
from nose.tools import *  # noqa: F403
```

to be explicit about what rule we're ignoring.